osu.Game/Overlays/Settings/Sections/Input/TabletSettings.cs

```diff
@@ -73,12 +77,44 @@ private void load()
                     Origin = Anchor.TopCentre,
                     Current = tabletHandler.Enabled
                 },
-                noTabletMessage = new OsuSpriteText
+                noTabletMessage = new FillFlowContainer
                 {
-                    Text = TabletSettingsStrings.NoTabletDetected,
-                    Anchor = Anchor.TopCentre,
-                    Origin = Anchor.TopCentre,
-                    Padding = new MarginPadding { Horizontal = SettingsPanel.CONTENT_MARGINS }
+                    RelativeSizeAxes = Axes.X,
+                    AutoSizeAxes = Axes.Y,
+                    Direction = FillDirection.Vertical,
+                    Padding = new MarginPadding { Horizontal = SettingsPanel.CONTENT_MARGINS },
+                    Spacing = new Vector2(5f),
+                    Children = new Drawable[]
+                    {
+                        new OsuSpriteText
+                        {
+                            Anchor = Anchor.TopCentre,
+                            Origin = Anchor.TopCentre,
+                            Text = TabletSettingsStrings.NoTabletDetected,
+                        },
+                        new LinkFlowContainer
```

Comment:
May want to make a class for this (to consolidate the visual behaviour with `SettingsItem.WarningText`).

Reply:
`WarningText` is currently only a string, that's why I backed out from refactoring it to pass a `TextFlowContainer` just yet.

Reply:
I meant the backing drawable that's displaying that string. Just making it a custom `LinkFlowContainer` with the correct styling applied. I think that's feasible?

Reply:
Aha, yeah should be feasible.